examples/vad_transcriber/wavTranscription.md

```diff
@@ -63,3 +104,11 @@ In such a scenario, the GUI tool will not work. The following steps is known to
 (venv) ~/Deepspeech/examples/vad_transcriber$ python3 audioTranscript_gui.py
 
 ```
+#### 2.2 Known Bugs
+#####  Could not load modal with error code X
```

Comment:
Actually, `X` does gives the answer: https://deepspeech.readthedocs.io/en/latest/Error-Codes.html

Reply:
You are right sorry, would it be a good Idea to add a link to the readthedocs documentation at the bottom of the document?
Any chance to add the decimal value of the error to the documentation?

Reply:
There's already a link at the top :)